src/components/exercises/LetterRotated.js

```diff
@@ -9,29 +9,41 @@ class LettersRotated extends Component {
     super(props);
 
     this.state = {
-      highlighted: null
+      highlighted: [false, false, false, false, false]
     };
   }
 
-  createLetterButton = index => {
-    const { angle, letters, rotated } = this.props;
+  allAngles(size, angles, rotated) {
+    var ret = new Array(size), index;
+    for (var i = 0; i < size; i++) {
+      index = rotated.indexOf(i);
+      ret[i] = index !== -1 ? angles[index] : '0deg';
+    }
+    return ret;
```

Comment:
I'm thinking about changing the API a little bit since this method isn't so readable, IMHO. How about this:

```.js
getTransform = (index) => {
  const angleIndex = this.props.rotated.indexOf(index)
  return angleIndex !== -1 ? [{ rotate: this.props.angles[angleIndex] }] : [];
}
```

What do you think about that, @vhaller?